tests/integration/v3_watch_restore_test.go

```diff
@@ -81,14 +87,30 @@ func TestV3WatchRestoreSnapshotUnsync(t *testing.T) {
 	}
 
 	clus.Members[0].InjectPartition(t, clus.Members[1:]...)
-	initialLead := clus.WaitMembersForLeader(t, clus.Members[1:])
+	initialLead := clus.WaitMembersForLeader(t, clus.Members[1:]) + 1
 	t.Logf("elected lead: %v", clus.Members[initialLead].Server.MemberId())
 	t.Logf("sleeping for 2 seconds")
 	time.Sleep(2 * time.Second)
 	t.Logf("sleeping for 2 seconds DONE")
 
 	kvc := integration.ToGRPC(clus.Client(1)).KV
 
+	// NOTE: In 3 members cluster, after initial lead has been elected,
+	// there are 3xConfChange + 3xMemberAttrSet + 1xClusterVersionSet logs.
```

Comment:
This isn't correct. Note that when starting a new cluster with 3 members, each member will apply 3 ConfChange directly at the beginning before a leader is elected. 

Please read https://github.com/etcd-io/raft/blob/918455b897764d8e4d12af70657280365ffaaa04/bootstrap.go#L76-L78

Reply:
Thanks. I will update it later.

Reply:
Updated!